hardhat.config.ts

```diff
@@ -503,6 +503,14 @@ export default {
           browserURL: 'https://modescan.io',
         },
       },
+      {
+        network: 'avalanche',
+        chainId: 43114,
+        urls: {
+          apiURL: 'https://api.snowtrace.io/api',
+          browserURL: 'https://snowtrace.io',
+        },
+      },
```

Comment:
```suggestion
```

this is already present in `chain-config.ts` inside hardhat's libraries; no need for this.